src/app/app-platform/ContentAppPlatform.h

```diff
@@ -38,6 +38,17 @@ using BindingListType = chip::app::Clusters::Binding::Attributes::Binding::TypeI
 namespace chip {
 namespace AppPlatform {
 
+// The AppPlatform overrides emberAfExternalAttributeReadCallback to handle external attribute reads for ContentApps.
+// This callback can be used to handle external attribute reads for attributes belonging to static endpoints.
+EmberAfStatus AppPlatformAfExternalAttributeReadCallback(EndpointId endpoint, ClusterId clusterId,
```

Comment:
Why the "Af" in the name?  Probably better to take that out, here and for the write callback.

Reply:
Removed in latest commit, please review. @bzbarsky-apple